src/github.com/getlantern/flashlight/main/settings.go

```diff
@@ -27,15 +29,18 @@ var (
 
 // Settings is a struct of all settings unique to this particular Lantern instance.
 type Settings struct {
-	Version      string
-	BuildDate    string
-	RevisionDate string
-	AutoReport   bool
-	AutoLaunch   bool
-	ProxyAll     bool
-	SystemProxy  bool
-
-	sync.RWMutex
+	UserId    int    `json:"user_id,omitempty"`
+	UserToken string `json:"user_token,omitempty"`
+
+	Version      string `json:"version"`
+	BuildDate    string `json:"build_date"`
+	RevisionDate string `json:"revision_date"`
+	AutoReport   bool   `json:"auto_report"`
+	AutoLaunch   bool   `json:"auto_launch"`
+	ProxyAll     bool   `json:"proxy_all"`
+	SystemProxy  bool   `json:"system_proxy"`
```

Comment:
yeah it seems like camel case is overall the most widely used standard for js (see, for example, http://www.w3schools.com/js/js_conventions.asp and https://google.github.io/styleguide/javascriptguide.xml), which I'm a little surprised Go doesn't just automatically convert to, but it also looks like on the frontend in a lot of cases you're converting *from camel case* to using underscores (https://github.com/getlantern/lantern/pull/3788/files#diff-8a1026309248e12a3231dfa77eb4755eR81), and there are only a few places where we actually have caps at the beginning. Not a huge deal, but just seems like a lot of code to change when in most cases we're actually moving to a *less* standard naming scheme!